packages/next/src/build/webpack/loaders/metadata/discover.ts

```diff
@@ -20,6 +28,15 @@ const staticAssetIconsImage = {
     filename: 'favicon',
     extensions: ['ico'],
   },
+  opengraph: {
+    filename: 'opengraph-image',
+    // TODO: share basic image extensions with next/image related utils
+    extensions: ['jpg', 'jpeg', 'png', 'svg', 'avif', 'webp', 'svg'],
+  },
+  twitter: {
+    filename: 'twitter-image',
+    extensions: ['jpg', 'jpeg', 'png', 'svg', 'avif', 'webp', 'svg'],
```

Comment:
I don't think it's allowed to use formats other than JPG, PNG and GIF for OG images or Twitter images, unless we are converting formats automatically.